packages/sdk/src/clients/multicall/client.e2e.ts

```diff
@@ -26,11 +31,27 @@ describe("multicall", function () {
         callData: empClient.interface.encodeFunctionData(call),
       };
     });
-    const response = await client.callStatic.aggregate(multicalls);
+    const response = await clientV1.callStatic.aggregate(multicalls);
     const decoded = calls.map((call: any, i: number) => {
       const result = response.returnData[i];
       return empClient.interface.decodeFunctionResult(call, result);
     });
     assert.equal(decoded.length, calls.length);
   });
+
+  test("multicall2 on emp", async function () {
+    const calls = ["priceIdentifier", "tokenCurrency", "collateralCurrency"];
+    const multicalls = calls.map((call: any) => {
+      return {
+        target: empAddress,
+        callData: empClient.interface.encodeFunctionData(call),
+      };
+    });
+    const response = await clientV2.callStatic.tryBlockAndAggregate(false, multicalls);
+    const decoded = calls.map((call: any, i: number) => {
+      const result = response.returnData[i];
+      return empClient.interface.decodeFunctionResult(call, result[1]);
+    });
+    assert.equal(decoded.length, calls.length);
+  });
```

Comment:
It would also be good to see a test in multicall2 where one of the calls in the list actually fails while others succeed